src/launchpad/size/models/apple.py

```diff
@@ -52,6 +52,12 @@ def install_size(self) -> int:
         return self.file_analysis.total_size  # TODO: Implement install size calculation
 
 
+class LocalizedStringInsightResult(BaseInsightResult):
+    """Results from localized string analysis."""
+
+    files: List[FileInfo] = Field(..., description="Localized strings files exceeding 100KB threshold")
```

Comment:
nit: the comment is slightly wrong since it's whether the total size exceeds the threshold, not per file